test/Java8andUp/playlist.xml

```diff
@@ -754,6 +754,64 @@
 		</subsets>
 	</test>
 
+	<test>
+		<featureIds>
+			<featureId>J9VMInternals</featureId>
+		</featureIds>
+		<testCaseName>J9VMInternals_Test_SE90</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(MKDIR) -p $(REPORTDIR); \
+	cd $(REPORTDIR); \
+	$(JAVA_COMMAND) $(JVM_OPTIONS) -verbose:stacktrace \
+	-Djava.security.policy=$(Q)$(TEST_RESROOT)$(D)java.policy$(Q) \
+	-Xbootclasspath/a:$(TEST_RESROOT)$(D)TestResources.jar \
+	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(P)$(TEST_RESROOT)$(D)TestResources.jar$(P)$(LIB_DIR)$(D)asm-all.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+	-testnames JCL_TEST_Java-Internals \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<platformRequirements>^vm.hrt</platformRequirements>
+		<tags>
+			<tag>sanity</tag>
+		</tags>
```

Comment:
Please update playlist.xml as we have https://github.com/eclipse/openj9/pull/1036 merged in. For example,
```
<levels>
   <level>sanity</level>
</levels>
<groups>
   <group>functional</group>
</groups>
```

Reply:
Sorry, I noticed that this is already merged. Please still update the playlist in another pr. With current version of testkitgen, it will ignore the tags and test will run in extended.